hydra-tui/src/Hydra/TUI.hs

```diff
@@ -357,49 +353,57 @@ handleCommitEvent Client{sendInput, myAddress} CardanoClient{queryUtxoByAddress}
     Dialog title form submit
    where
     title = "Select UTXO to commit"
-    -- TODO: This should really be a radio field, because we want to only allow
-    -- one UTXO entry to be committed.
     form = newForm (utxoCheckboxField u) ((,False) <$> u)
```

Comment:
Why not use the radio field as indicated? Couldn't find it mentioned in the logbook, thus I'm asking here.

Reply:
It's because we added a guard to limit the number of selected items to 0 or 1. Radio fields require exactly 1 item selected but it's fine to _not commit_ any UTXO.